facts.d/example_control_repo.yaml

```diff
@@ -0,0 +1,5 @@
+# This is an example of how to use the control_repo_path custom fact
+# for deploying your own control repo instead of the default.
+#
+#control_repo_path: 'https://github.com/thespain/control-repo.git'
```

Comment:
For the sake of the sake of the example, please replace `thespain` with `your-name-here`